client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/rates-step/index.js

```diff
@@ -98,6 +98,11 @@ const getRatesStatus = ( { retrievalInProgress, errors, available, form } ) => {
 const showCheckoutShippingInfo = props => {
 	const { shippingMethod, shippingCost, translate } = props;
 
+	// Use the DOM in order to convert HTML entities into text
```

Comment:
Could clarify "the DOM" in this comment so it's clear this code isn't actually manipulating _the_ DOM – maybe refer to "a DOM element" or "the DOM API"?

Reply:
👍 

I'm used to calling it the DOM tree and I did not even realise that DOM might be misleading in this context.